Description: Remove the -N RTS flag
 This flag is not supported on architectures with no SMP.
Author: Ilias Tsitsimpis <iliastsi@debian.org>
Bug: https://github.com/haskell-crypto/cryptonite-conduit/issues/3
Bug-Debian: https://bugs.debian.org/880429
---

Index: b/cryptonite-conduit.cabal
===================================================================
--- a/cryptonite-conduit.cabal
+++ b/cryptonite-conduit.cabal
@@ -45,7 +45,7 @@ test-suite cryptonite-conduit-test
                      , memory
                      , tasty
                      , tasty-hunit
-  ghc-options:         -threaded -rtsopts -with-rtsopts=-N -W -Wall
+  ghc-options:         -threaded -rtsopts -W -Wall
 
 source-repository head
   type: git
